src/AppBundle/Resources/views/search/search.html.twig

```diff
@@ -3,6 +3,12 @@
 {% block body %}
     <div id="search-page" v-cloak>
 
+        {# This block is used by :
```

Comment:
This block might be used in:

Reply:
ben pour le coup, il est utilisé :thinking:

Reply:
pas forcément dan company.html.twig

Reply:
https://github.com/wizaplace/starterkit/pull/216/files/32cbc54163ca70f68be05c012bade391400d6497#diff-f9b5613b9b79b8a0bd403add30dfa5a3R3

Si, il l'est forcément, mais on veut peut être un bloc supplémentaire pour afficher les reviews en bas (je me suis peut être mal exprimé dans l'autre comment pour le coup ?